src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/JwtAccessTokenBuilder.kt

```diff
@@ -59,6 +60,7 @@ fun buildAccessToken(
         "auth_source" to "nomis",
         "user_uuid" to UUID.randomUUID(),
         "name" to displayName,
+        "client_id" to clientId,
       ),
```

Comment:
Our JWT tokens that our integration tests use need a client_id claim because the interceptor above reads that claim and adds it to the custom dimensions
(the real JWT we get from hmpps-auth will contain a client_id claim)